src/Azure.Functions.Cli/StaticResources/customHandler.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "description": {
+    "defaultExecutablePath": "",
+    "workingDirectory": "",
+    "arguments": [ ]
+  },
+  "enableForwardingHttpRequest": true
```

Comment:
Wonder if we should default `enableForwardingHttpRequest` to `false`.

Reply:
by default `enableForwardingHttpRequest` is `false`. Ok to omit this altogether?

Reply:
IMO we should keep it there and set to `false`. It makes it easy to change, and also makes the setting discoverable.

I was actually wondering if we should add the extensions.http.routePrefix by default as well so custom handlers customers can change it from `"api"` to `""` easily. I think that might be a bit much though.

Reply:
@yojagad - let's set `enableForwardingHttpRequest` explicilty to `false`

Lets skip introducing route prefix here as it is specific to httpTrigger and host.json for other languages do not have it either.

Reply:
Based on the earlier conversation we had around supportability, etc, I removed this setting altogether. Let me know if you still think I should add it back with it set to false

Reply:
I’m okay with taking it out for now.